app/controllers/file_uploads_controller.rb

```diff
@@ -23,6 +23,20 @@ def upload
     @file    = @product.stored_files.new(:file_type => params[:file_type])
   end
 
+  # GET /facilities/:facility_id/:product/:product_id/files/:file_type/:id
+  def download
```

Comment:
I think if you change the before_filters to:
```
load_and_authorize_resource :class => StoredFile, :except => [:uploader_create, :download]
```
That should solve your 403 problem (I see no reason why any of these files need to be authorized).